docs/quickstart/multicluster.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Multicluster setup
+weight: 200
+---
+
+The following procedure shows you how to collect metrics from a peer cluster to an existing One Eye deployment. The cluster running the One Eye deployment is called the **observer cluster**, the new cluster you are collecting metrics from is called the **peer cluster**. For details on how One Eye collects metrics from multiple clusters, see {{% xref "/docs/one-eye/multicluster/_index.md" %}}.
+
+## Prerequisites
+
+- You have completed the steps described in {{% xref "/docs/one-eye/quickstart/_index.md" %}} and you have a working One Eye deployment.
+- You have another cluster that will be the peer cluster.
+
+{{< include-headless "warning-prometheus-label.md" "one-eye" >}}
+
+{{< include-headless "multicluster-metrics-limitations.md" "one-eye" >}}
+
+## Steps
+
+> Note: The following procedure requires you to switch between the Kubernetes context of the observer and the peer cluster. A convenient way to switch between contexts is to use the [kubectx tool](https://github.com/ahmetb/kubectx).
+
+<!-- FIXME: Include a kubectx-less command? -->
+
+1. Obtain the name of the peer cluster, for example, by getting it from the current context of the peer's kubeconfig. 
+
+    On the peer cluster, run the following command.
+
+    ```bash
+    kubectx $PEER_CONTEXT
+    export PEER_ENDPOINT=$(kubectl config current-context | cut -d '@' -f 2)
+    ```
+
+1. Switch to the observer cluster.
+
+    ```bash
+    kubectx $OBSERVER_CONTEXT
+    ```
+
+1. If not already installed, install cert-manager and thanos-operator components on the observer cluster.
+
+    ```bash
+    one-eye cert-manager install --update
+    one-eye thanos install --operator-only --update
+    ```
+
+1. Create a secret that will be used to establish trust between the observer and the peer clusters. The following steps show you how to use the same certificate on both clusters.
+    <!-- FIXME: Mention other possibilities
+    > pepov mentioned that secrets from an external PKI like vault can be used - how?
+    > what if the clusters are in a service mesh? -->
+
+    1. Create a self-signed CA and a certificate.
+
+        ```bash
+        cat <<EOF | kubectl apply -f-
+        apiVersion: cert-manager.io/v1
+        kind: Issuer
+        metadata:
+          name: selfsigned
+        spec:
+          selfSigned: {}
+        EOF
+        ```
+
+    1. Create a Kubernetes secret for the certificate with proper labels. The labels are needed so the Thanos operator can find the secret and attach it to the ThanosPeer custom resource.
+
+        ```bash
+        cat <<EOF | kubectl apply -f-
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: peer-tls
+          labels:
+            monitoring.banzaicloud.io/thanospeer: ${PEER_ENDPOINT}
+            monitoring.banzaicloud.io/thanospeer-ca: ${PEER_ENDPOINT}
```

Comment:
we should highlight the fact that these labels are important for the thanospeer to be able to use them automatically